src/Content/Backend/Solution/Monaco.Template.Backend.Domain.Tests/Factories/Entities/DocumentFactory.cs

```diff
@@ -1,18 +1,19 @@
 ﻿using AutoFixture;
+using Monaco.Template.Backend.Common.Tests;
 using Monaco.Template.Backend.Domain.Model;
 using Moq;
 
-namespace Monaco.Template.Backend.Common.Tests.Factories.Entities;
+namespace Monaco.Template.Backend.Domain.Tests.Factories.Entities;
 
 public class DocumentFactory
 {
 	public static Document Create() =>
-		new Fixture().RegisterDocument()
-					 .Create<Document>();
+		FixtureFactory.Create(f => f.RegisterDocument())
+					  .Create<Document>();
 
 	public static IEnumerable<Document> CreateMany() =>
-		new Fixture().RegisterDocument()
-					 .CreateMany<Document>();
+		FixtureFactory.Create(f => f.RegisterDocument())
```

Comment:
Should this be using the `RegisterDocumentMock` method?

Reply:
Looking back at this, I think we don't really need the Mock version... Because the entity already allows to initialize it with an Id value, so there isn't really a need to mock it at all.

Reply:
Fixed.